docs/using/security-non-events.md

```diff
@@ -0,0 +1,58 @@
+
+## Wordpress
+
+The Wordpress app on Sandstorm is used to publish public web sites. However, the app does not respond to individual page views of that site. Instead, the app generates static content which it hands off to Sandstorm, and Sandstorm serves the site statically. The site administrator accesses Wordpress's administrative interface through Sandstorm like any other app, but visitors to the public site visit a separate hostname where they don't see Sandstorm at all. The site owner may share access to the edit UI but would never share this access publicly.
+
+As a result of this model, there is no way to exploit a bug in Wordpress on Sandstorm if you only have access to the published site, because there is no way for you to cause any Wordpress code to execute at all. The site owner must explicitly share access to the admin interface with you before you can interact with the app.
+
+Additionally, because the Wordpress admin interface is served through Sandstorm [on a random, unguessable hostname](https://docs.sandstorm.io/en/latest/developing/security-practices/#client-sandboxing), CSRF attacks are far more difficult to carry out against this interface. (They are not impossible, but they require a passive MITM of network traffic or other information leaks to carry out.)
+
+This model has a down side: comments are currently not supported. However, with some engineering effort, comments could be re-enabled while maintaining strong security. To accomplish this, the public-facing web site would use client-side Javascript to communicate with an HTTP API exported by the Sandstorm app whenever a user posts a comment. In order to prevent vulnerabilities in the comment-handling code from threatening core site administration, comments could be managed by a separate app that runs in a separate grain (container) from the core site. With the right design, there is no need for information about comments to flow into the core site management, so vulnerabilities in the comment infrastructure would at worst allow an attacker to deface other users' comments, not the whole web site.
+
+[Wordpress has reported many vulnerabilities](http://www.cvedetails.com/vulnerability-list/vendor_id-2337/product_id-4096/). Below, we cover all Wordpress CVEs scored with a severity of 6 or more in 2014 and 2015.
+
+* [CVE-2015-5731](http://www.cvedetails.com/cve/CVE-2015-5731/): CSRF allows exercising admin's ability to lock posts. Sandstorm provides additional CSRF protection as described above.
```

Comment:
Suggest "CSRF" => "A CSRF vulnerability" for clarity; I can do that in a separate pull request on top